packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/process-dq.test.ts

```diff
@@ -44,4 +44,19 @@ describe("processDQResponse", () => {
     });
     expect(response.operationOutcome?.issue[0]?.code).toEqual("XDSRegistryError");
   });
+
+  it("should process response that is not a string correctly", async () => {
+    const randomResponse = "This is a bad response and is not xml";
+
+    const response = processDQResponse({
+      dqResponse: {
+        success: true,
+        response: randomResponse,
+        gateway: outboundDQRequest.gateway,
+        outboundRequest: outboundDQRequest,
+      },
+    });
+    expect(response.operationOutcome).toBeDefined();
```

Comment:
Prefer `toBeTruthy` as `toBeDefined` passes even if the value is `null`.

https://jestjs.io/docs/expect#tobedefined

https://jestjs.io/docs/expect#tobetruthy